app/src/main/java/com/vero/tasky/agenda/data/remote/repository/AgendaRepositoryImpl.kt

```diff
@@ -0,0 +1,36 @@
+package com.vero.tasky.agenda.data.remote.repository
+
+import com.vero.tasky.agenda.data.remote.network.AgendaApi
+import com.vero.tasky.agenda.data.remote.network.request.SyncAgendaRequest
+import com.vero.tasky.agenda.domain.model.Agenda
+import com.vero.tasky.agenda.domain.repository.AgendaRepository
+import com.vero.tasky.core.data.remote.ApiCallHandler
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.flow
+
+class AgendaRepositoryImpl(
+    private val api: AgendaApi
+) : AgendaRepository {
+
+    override suspend fun getAgendaForDay(timezone: String, time: Long): Flow<Result<Agenda>> = flow {
+    }
+
+    override suspend fun syncAgenda(
+        deletedEventsIds: List<String>?,
```

Comment:
Something I personally do on all my apps is

Never have Null lists
If a null is needed for a list, I just send it empty

Why? Avoid null errors, + you just cover it by checking if a null is not empty (for loops, while loops, etc already cover this)